tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -168,3 +178,103 @@ have unreleased changes, publish these packages to pub.
+
+2. Create a new branch for your cherry pick release. 
+   ```
+   git checkout -b 2.29.1
+   ```
+
+3. Cherry pick the commit(s) you want in this cherry-pick release, and bump the DevTools version number:
+   ```
+   git cherry-pick <commit>
+   devtools_tool update-version auto -t patch
+   ```
+
+4. Commit your changes and push to the `upstream` remote.
+   ```
+   git add .
+   git commit -m "Prepare cherry-pick release - DevTools 2.29.1"
+   git push upstream 2.29.1
+   ```
+
+Once you are completely satisfied with your changes, create a tag for this cherry-pick release. This
+tag is purely for the management and health of the `flutter/devtools` repo; it will not be used for the
+remainder of the cherry-pick release process. From the branch your are already on (e.g. `2.29.1`), run
+the following:
+   ```
+   tool/tag_version.sh
+   ```
+
+**To move on to the next step, you will need to take note of two values:**
+1) The name of the DevTools branch you just created (e.g. `2.29.1`)
+2) The commit hash that is at the tip of this branch (see https://github.com/flutter/devtools/branches).
+
+### Manually run the DevTools Builder
+
+Follow the instructions at [go/dart-engprod/devtools.md#cherry-picks](go/dart-engprod/devtools.md#cherry-picks)
+to trigger the DevTools builder.
+
+### Create the cherry-pick CL in the Dart SDK
+
+Checkout the Dart SDK branch you want to perform the cherry-pick on top of (e.g. `stable` or `beta`),
+and create a new branch:
+
+```
+git new-branch --upstream origin/<stable or beta> cherry-pick-devtools
+```
+
+Edit the "devtools_rev" entry in the Dart SDK [DEPS](https://github.com/dart-lang/sdk/blob/main/DEPS#L104) file
+to point to the cherry-pick release hash (the commit at the tip of the cherry-pick branch you created above).
+
+Commit your changes and upload your CL:
+```
+git add .
+git commit -m "Cherry-pick DevTools 2.29.1"
+git cl upload -s
+```
+
+### Create the merge commit in the `flutter/devtools` repo
+
+This step can be performed while you are waiting for the Dart SDK cherry-pick CL to be reviewed. In order
+to ensure that the cherry-picked DevTools commit does not get GC'ed, we need to perform a merge commit
+from the branch we just created (e.g. `2.29.1`) onto the `flutter/devtools` protected branch (`master`).
+
+1. Create a pull request in the GitHub UI from the cherry-pick branch to the `master` branch.
+   - Navigate to https://github.com/flutter/devtools/compare/\<cherry-pick-branch>, where \<cherry-pick-branch>
+      is the branch you pushed up to the `upstream` remote with the cherry-picked commit(s) (e.g. `2.29.1`).
+   - Click "Create pull request"
+   - Resolve any merge conflicts by **accepting whatever is on master**. Once you do this, the PR should show
+    changes in **zero** lines of code.
+   - Ask a member of the DevTools team for review, but **DO NOT** squash and merge yet.
+
+2. Contact a member of the Dash team who has Admin access to the `flutter/devtools` repository settings
```

Comment:
This process may not be scalable as there are only three admins, if force push is what is required we can probably grant force push to a group of people from devtools to avoid the dependency on admins and then remove steps 2,3 and 4.

Reply:
If we could grant admin access for modifying this setting to @jacob314 and myself for the `flutter/devtools` repo, that should suffice to remove the bottleneck.

Reply:
> This process may not be scalable as there are only three admins, if force push is what is required we can probably grant force push to a group of people from devtools to avoid the dependency on admins and then remove steps 2,3 and 4.

No force push needed, it's just about enabling merging a PR with a merge commit (one of the three options, usually we only enable squash, but leave rebase and merge disabled). But for this to work it's essential that it's a merge commit.

Note that flutter/devtools is probably the only flutter repo we'll need this for. It's only needed for dependencies in the Dart SDK DEPS (though it might be a good idea for flutter/engine DEPS, too).

Reply:
I think I'm missing some context: why does this need to be a merge commit?

Reply:
Because only a merge commit has multiple parents and keeps the cherry-pick branch reachable from the main branch.

Reply:
ahh, so this would be merging release branches INTO main? This makes sense.

Reply:
> If we could grant admin access for modifying this setting to @jacob314 and myself for the flutter/devtools repo, that should suffice to remove the bottleneck.

@godofredoc do you have further guidance on this? Or should we just leave the three admins as the gatekeepers for now?